gymnasium/envs/box2d/bipedal_walker.py

```diff
@@ -617,7 +617,7 @@ def render(self):
         if self.render_mode is None:
             assert self.spec is not None
             gym.logger.warn(
-                "You are calling render method without specifying any render mode. "
+                "You are calling render method without specifying any render mode."
```

Comment:
This is correct, revert. The extra space is necessary for the next line

Reply:
good catch, just rolled back